sdk/purview/azure-purview-datamap/tsp-location.yaml

```diff
@@ -0,0 +1,4 @@
+directory: specification/purview/Azure.Analytics.Purview.DataMap
+commit: 93813542ad3f0cdd3bc31c03dae2ff3f8d802254
+repo: Azure/azure-rest-api-specs
+additionalDirectories:
```

Comment:
Please follow https://github.com/Azure/azure-sdk-for-python/wiki/What-to-do-after-generating-the-SDK-code-with-codegen#createupdate-the-ciyml to update ci.yml

Reply:
updated